src/org-units-selector/OrgUnitsSelector.jsx

```diff
@@ -99,7 +99,7 @@ export default class OrgUnitsSelector extends React.Component {
     }
 
     getRoots({ filter } = {}) {
-        const { api, listParams, rootIds } = this.props;
+        const { api, listParams, rootIds, selectableLevels } = this.props;
```

Comment:
I guess ``selectableLevels`` comes from d2-ui, anyway since it's not a disable but hiding operartion, would it make sense something like ``visibleLevels``?

Reply:
Indeed we saw the naming/logic is confusing, but were in a hurry, so I opened a separate issue to do somewhere in time: https://github.com/EyeSeeTea/d2-ui-components/issues/134

Reply:
Great then